src/main/java/com/example/repository/TicketRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.repository;
+
+
+import com.example.models.Ticket;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface TicketRepository extends JpaRepository<Ticket, Long> {
```

Comment:
add at least 1 method for `TicketRepository` or `UserRepository` with custom query